pages/treeherder.py

```diff
@@ -51,6 +59,42 @@ def open_perfherder_page(self):
         from perfherder import PerfherderPage
         return PerfherderPage(self.base_url, self.selenium).wait_for_page_to_load()
 
+    def select_next_job(self):
+        el = self.selenium.find_element(*self._resultset_locator)
+        Wait(self.selenium, self.timeout).until(EC.visibility_of(el))
+        el.send_keys(Keys.ARROW_RIGHT)
+        Wait(self.selenium, self.timeout).until(lambda s: self.job_result_status)
+        return self.selenium.find_element(*self._next_job_title).get_attribute('title')
+
+    def add_selected_job_to_pinboard(self):
+        el = self.selenium.find_element(*self._resultset_locator)
+        Wait(self.selenium, self.timeout).until(EC.visibility_of(el))
+        el.send_keys(Keys.SPACE)
+        Wait(self.selenium, self.timeout).until(lambda s: self.is_pinboard_open)
```

Comment:
I think we'd want to wait for the new job to be pinned, as the pinboard may already be open. Perhaps we could wait for the number of pinned jobs to increase by one?